taittirIyam/sUtram/baudhAyanaH/brahma-karma-samuchchayaH/4_pitRmedhakANDam/05_pitRmedhe_prAyashchittAni.md

```diff
@@ -37,14 +37,35 @@ title = "०५ पितृमेधे प्रायश्चित्ता
  इति सङ्कल्प्याऽग्निमपसव्यं परिषिञ्च्य स्रुचि चतुर्गृहीतं गृहीत्वोत्तरतो दक्षिणामुख उपविश्य सकृज्जुहोति - 'मृत्योरधिष्ठानाय स्वाहा' । मृत्योरधिष्ठानायेदं ॥
  ततः पृथक्पृथक्चतुर्ग्रहीतेन षण्मन्त्रैर्जुहोति -
 
- >ॐ ता सूर्याचन्द्रमसा जातु देवयोः स्वाहा । सर्वत्र सूर्याचन्द्रमोभ्यामिदम् ॥ उभावन्तौ हतामती स्वाहा ॥ पती धुमत० महिव्रता स्वाहा ॥ विश्ववपरी० सन्दिवँ स्वाहा ॥ अस्य श्रवो० विचर्तुरँ स्वाहा ॥ पूर्वापरं जायते पुनः स्वाहा
- इति षभिर्तुत्वाऽग्निं | पुनरपसव्यं परिषिच्याऽमुकगोत्रस्य प्रेतस्य दक्षिणायनमरणजनितसमस्तदोषपरिहारार्थं उत्तरायणमरणसम्पत्तिद्वारा उत्तमलोकप्राप्त्यर्थं अद्यप्रभृत्युत्तरायणाऽऽरम्भदिनपर्यन्तं भविष्यदौपासनाहुतिपरिमितद्रव्यं तन्निष्क्रयहिरण्यं वा आशौचान्ते दास्ये इति सङ्कल्प्याऽऽशौचान्ते दद्यात् ॥
+ > ॐ ता सूर्याचन्द्रमसा जातु देवयोः स्वाहा । सर्वत्र सूर्याचन्द्रमोभ्यामिदम् ॥ उभावन्तौ हतामती स्वाहा ॥ पती धुमत् महिव्रता स्वाहा ॥ विश्ववपरी० सन्दिवँ स्वाहा ॥ अस्य श्रवो० विचर्तुरँ स्वाहा ॥ पूर्वापरं जायते पुनः स्वाहा
+ इति षड्भिर्हुत्वाऽग्निं पुनरपसव्यं परिषिञ्च्याऽमुकगोत्रस्य प्रेतस्य दक्षिणायनमरणजनितसमस्तदोषपरिहारार्थं उत्तरायणमरणसम्पत्तिद्वारा उत्तमलोकप्राप्त्यर्थं अद्यप्रभृत्युत्तरायणाऽऽरम्भदिनपर्यन्तं भविष्यदौपासनाहुतिपरिमितद्रव्यं तन्निष्क्रयहिरण्यं वा आशौचान्ते दास्ये इति सङ्कल्प्याऽऽशौचान्ते दद्यात् ॥
 ## कृष्णपक्षमरणे
-कृष्णपक्षमरणप्रायश्चित्तम् ॥ “अमुकगोत्रस्य प्रेतस्य कृष्णपक्षमरणजनितसमस्तदोषनिवृत्त्या उत्तमलोकप्राप्त्यर्थ 'ता सूर्याचन्द्रमसा' इत्यादि षण्मन्त्रैः प्रायश्चित्ताज्याहुतीोष्ये " इति सङ्कल्प्याऽदौ चतुर्ग्रहीतेन — मृत्योरधिचानाय स्वाहा' (मृत्योर धिष्ठानायेदं ) इति हुत्वा पृथक्पृथक्चतुर्ग्रहीतेन ता सूर्यादिषडाहुतीश्च हुत्वाऽमुकस्य प्रेतस्य शुक्लपक्षमरणसम्पत्यर्थमद्यप्रभृति | दर्शान्तान्सायमौपासनहोमान्शुक्ल प्रतिपत्पर्यन्तान प्रातरौपासनहोमांश्चाऽपकृष्य तन्त्रण तण्डुलैोष्ये । दर्शस्थालीपाकसंपत्त्यर्थ
+कृष्णपक्षमरणप्रायश्चित्तम् ॥ 
 
-पूर्णाहुतिश्च होष्ये इति सङ्कल्प्य तण्डुलान् संस्कृत्याऽग्निमपसव्यं परिषिच्यैकां समिधमभ्याधाय स्थाल्यां दर्शपर्यन्तं वृद्धिक्षयानु ही सारेण भविष्यत्तिथिसङ्ख्याकं सकृत्सकृत्तण्डुलमुष्टिं गृहीत्वा द्वेधा विभज्य प्रथमभागं 'अग्नये स्वाहा' द्वितीयभाग ' प्रजापतये
+> अमुकगोत्रस्य प्रेतस्य कृष्णपक्षमरणजनितसमस्तदोषनिवृत्त्या उत्तमलोकप्राप्त्यर्थं
+>> ता सूर्याचन्द्रमसा
+इत्यादि षण्मन्त्रैः प्रायश्चित्ताज्याहुतीर्होष्ये
 
-स्वाहा' इति सकृत्सकृदुत्वा पुनः शुक्ल प्रतिपत्पर्यन्तातथिसङ्ख्यया सकृत्सवगृहीत्वा द्वेधा विभज्य प्रथमं ' सूर्याय स्वाहा द्वितीयं 'प्रजापतये स्वाहा' इति सकृत्सकृज्जुहुयात् ॥ दर्शस्थालीपाकस्थाने चुचि चतुर्गृहीतं गृहीत्वा ‘अग्निर्मूर्धा जिन्वतों ३ भवो यज्ञस्य हव्यवाहः स्वाहा' इति पूर्णाहुतिं जुहुयात् । मूर्धाग्नय इदं० ॥ पुनरग्निमपसन्यं परिषिश्चेत् ॥
+ इति सङ्कल्प्याऽदौ चतुर्गृहीतेन 
+
+ >> मृत्योरधिचानाय स्वाहा' (मृत्योर धिष्ठानायेदं )
```

Comment:
अत्र किमर्थं ">>" इति? 

तथा "मृत्योर धिष्ठानायेदं" इत्यत्र किमर्थं रिक्तस्थानम्?

Reply:
removed the space . used this as reference while giving >>
![image](https://user-images.githubusercontent.com/119562004/205890682-2ced2652-ee21-4c02-aebc-46e673b07470.png)

Reply:
किन्तु, अत्र "उद्धरणे उद्धरणम्" (quotation within quotation) इति नास्ति। तर्हि कुतः ">>"?

अपि च द्वितीयप्रश्नस्योपरि लिखितस्य+उत्तरं न लब्धम्।

Reply:
Done